src/core/RecyclerListView.tsx

```diff
@@ -147,7 +148,7 @@ export default class RecyclerListView<P extends RecyclerListViewProps, S extends
     private _tempDim: Dimension = { height: 0, width: 0 };
     private _initialOffset = 0;
     private _cachedLayouts?: Layout[];
-    private _scrollComponent: BaseScrollComponent | null = null;
+    private _scrollComponent: ScrollComponent | null = null;
```

Comment:
Why? RLV should be able to work with BaseScrollComponent, that's the contract

Reply:
Yea, I'd changed it when I'd exposed methods in ScrollComponent which needed to be used in RLV. Removed them later. Ideally though it should be ScrollComponent only for that reason?

Reply:
RLV should not have any change other than indices one.
Dude let's also deprecate onScroll and bind in the animated one